byterun/win32.c

```diff
@@ -861,3 +868,11 @@ CAMLexport caml_stat_string caml_stat_strdup_of_utf16(const wchar_t *s)
 
   return out;
 }
+
+CAMLexport char * win_utf8_string_of_utf16(const wchar_t * s)
+{
+  static char buf[1024];
```

Comment:
Is this use of global state correct in a multicore runtime? (Would the static buffer be copied by each runtime instance, which is fine, or is there a risk of race to a single buffer?)

Reply:
This was an alarm bell for me - I think @stedolan will confirm that we should be avoiding adding static buffers unless absolutely necessary.

Reply:
Well spotted, and yes, this would be bad in multicore. (I have some hacky scripts to detect such buffers, but they only work on ELF binaries. I'm not sure how to prevent them sneaking into windows-only codepaths, other than by vigilance)